conf/modules.config

```diff
@@ -319,22 +329,4 @@ process {
             pattern: "*{txt,vcf.gz}"
         ]
     }
-    withName: 'VCF_TO_FASTA' {
-        ext.when         = {  }
-        publishDir       = [
-            enabled: true,
-            mode: "${params.publish_dir_mode}",
-            path: { "${params.outdir}/combined/vcf-to-fasta" },
-            pattern: "*{fasta}"
-        ]
-    }
-    withName: 'VCF_CONSENSUS' {
-        ext.when         = {  }
-        publishDir       = [
-            enabled: true,
-            mode: "${params.publish_dir_mode}",
-            path: { "${params.outdir}/combined/consensus" },
-            pattern: "*{fasta.gz}"
-        ]
-    }
```

Comment:
I think it may have been that my forked repo was not up to date with the origin master branch. Sorry, I did not mean to delete them though. Thanks for all your help! - CJ